db/migrate/20160908100211_rename_programming_auto_grading_test_result_messages.rb

```diff
@@ -0,0 +1,18 @@
+class RenameProgrammingAutoGradingTestResultMessages < ActiveRecord::Migration
+  def change
+    add_column :course_assessment_answer_programming_auto_grading_test_results, :messages, :jsonb,
+               null: false, default: '{}'
+    serialize_existing_message
+    remove_column :course_assessment_answer_programming_auto_grading_test_results, :message
+  end
+
+  # message column currently stores test_case.error_message
+  def serialize_existing_message
+    Course::Assessment::Answer::ProgrammingAutoGradingTestResult.where.not(message: nil).
```

Comment:
@fonglh Still does not work, need to further add
`Course::Assessment::Answer::ProgrammingAutoGradingTestResult.reset_column_information` before this to let AR reload the column info.